### PR TITLE
Update DEVELOPERS.md

### PR DESCRIPTION
**Pull request by createyourpersonalaccount**
Be explicit about where docstrings go.

 

**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Clarifying the documentation on docstring placement. The `findMinMax()` example is actually documented in the source file but it is wrongly shown as documented in its declaration.

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by createyourpersonalaccount** [comment]
Is this style consistent across the project? Or do some Doxygen docstrings exist in header files too?

When the documentation is in the header files, they are distributed and it is helpful to the programmer to read them. When they are in the source files, the header files are leaner, so the installation is leaner; but the separate option to install documentation must be used.

**Comment by wargio** [comment]
> Is this style consistent across the project? Or do some Doxygen docstrings exist in header files too?

Doxygen comments are only in source files, not headers.

**Review by wargio — APPROVED** [review]

File: DEVELOPERS.md [hunk 1]
```diff
@@ -27,7 +29,7 @@ for more info. Example usage can be found [here](http://www.doxygen.nl/manual/do
  * Used to determine the min & max addresses of maps and
  * scale the ascii bar to the width of the terminal
  */
-static int findMinMax(RzList *maps, ut64 *min, ut64 *max, int skip, int width);
+static int findMinMax(RzList *maps, ut64 *min, ut64 *max, int skip, int width) { /* ... */ }
```
Comment (wargio) [inline DEVELOPERS.md]:
```suggestion
static int find_min_max(RzList *maps, ut64 *min, ut64 *max, int skip, int width) { /* ... */ }
```
This should not be in camel case.

**Comment by createyourpersonalaccount** [comment]
@wargio the camel case you corrected actually exists in the source code https://github.com/rizinorg/rizin/blob/71679b0800c657ad03fa374e989b2cd87433bd29/librz/core/cdebug.c#L550 I think you should revert that.

File: DEVELOPERS.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ doxygen C-style comments to improve documentation and comments in code.
 See the [Doxygen Manual](http://www.doxygen.nl/manual/index.html)
 for more info. Example usage can be found [here](http://www.doxygen.nl/manual/docblocks.html)
 
+Documentation goes into the source files (not the header files)
```
Comment (XVilka) [inline DEVELOPERS.md]: End sentences with dot please.

**Review by XVilka — CHANGES_REQUESTED** [review]

**Review by XVilka — APPROVED** [review]

**Comment by wargio** [comment]
> @wargio the camel case you corrected actually exists in the source code
> 
> https://github.com/rizinorg/rizin/blob/71679b0800c657ad03fa374e989b2cd87433bd29/librz/core/cdebug.c#L550
> I think you should revert that.

yes but still it should not be in camel case.

**Pull request merged**